### DISCUSSION TITLE
Displaying content with 'Show more...' or 'Show less...' functionality, limiting the display to 225 words, using HTML content and Line Clamp js.

### DISCUSSION THREAD

**Discussion by prachi-cp** [breezefront/community#242] [1. Help]
I would like an example of displaying content with 'Show more...' or 'Show less...' functionality, limiting the display to 225 words, using HTML content and Line Clamp js.

https://breezefront.com/docs/line-clamp

**Comment by vovayatsyuk** [comment]
Our lineClamp is based on css line-clamp property and can't limit the count of the words. Instead, it limits the number of rows.
